### PR TITLE
Add publish rpm script

### PR DESCRIPTION
**Pull request by sfc-gh-hyu**
I add some script for us to build and push rpm to internal yum repo. This script will be triggered by Jenkins job. All script is under firejail/snowflake directory, so there won't be conflict if we want to rebase with upstream firejail in the future.

### CODE REVIEW

File: snowflake/build_and_publish_rpm.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+#
+# Copyright (c) 2019 Snowflake Computing Inc. All right reserved.
```
Comment (sfc-gh-dbrown) [inline snowflake/build_and_publish_rpm.sh]: (c) 2020

File: snowflake/build_and_publish_rpm.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+TARGET_YUM_REPO=$1
+
+if [[ -z $TARGET_YUM_REPO ]]; then
+  echo "ERROR: {TARGET_YUM_REPO} not specified"
+  exit 1
+fi
+
+
+$THIS_DIR/../platform/rpm/mkrpm.sh firejail $FIREJAIL_SF_VERSION \
+    "--disable-userns --disable-contrib-install --disable-file-transfer --disable-x11 --disable-firetunnel"
```
Comment (sfc-gh-dbrown) [inline snowflake/build_and_publish_rpm.sh:17]: Why disable-userns?

File: snowflake/publish_rpm.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -ex
+#
+# Copyright (c) 2019 Snowflake Computing Inc. All right reserved.
```
Comment (sfc-gh-dbrown) [inline snowflake/publish_rpm.sh]: (c) 2020

**Review by sfc-gh-dbrown — COMMENTED** [review]
master is their dev branch, so it may have things that are not ready for prod.  I .. think.. (?) we should be applying this against release-0.9.62, or maybe have a release-sf branch?  Ultimately, the version.sh script should only exist on a branch.  That prevents deploying master by mistake.

**Comment by sfc-gh-hyu** [comment]
@sfc-gh-dbrown Yeah. I think it's probably best to maintain a release-sf branch and merge this PR into release-sf branch.

File: snowflake/build_and_publish_rpm.sh (hunk 2 above)
Comment (sfc-gh-hyu) [reply snowflake/build_and_publish_rpm.sh:17]: Default RPM build is disabling this. Do we need this?

**Pull request closed without merging**